contrib/report_builders/html_report_builder.py

```diff
@@ -0,0 +1,23 @@
+import os
+from typing import Any
+
+from jinja2 import Template, FileSystemLoader, Environment
+
+from contrib.descriptions import VulnDescriptionProvider
+from contrib.report_builders import JsonReportBuilder
+
+
+class JinjaHtmlReportBuilder(JsonReportBuilder):
+    def __init__(self, description_provider: VulnDescriptionProvider):
+        super().__init__(description_provider)
+        self.template_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'templates')
+        self.template_name = 'jinja2_report.html'
```

Comment:
I tested this and got the following error. Looks like jinja2_report.html doesnt exist?
```
Traceback (most recent call last):
  File "/output_report.py", line 81, in <module>
    main(*sys.argv[1:4], report_type=report_format)
  File "/output_report.py", line 64, in main
    builder = create_report_builder(report_type)
  File "/output_report.py", line 58, in create_report_builder
    return builder_map[report_type](provider)
  File "/output_report.py", line 50, in <lambda>
    'html': lambda p: JinjaHtmlReportBuilder(p),
  File "/contrib/report_builders/html_report_builder.py", line 15, in __init__
    self._read_template()  # type: Template
  File "/contrib/report_builders/html_report_builder.py", line 23, in _read_template
    self._template = template_env.get_template(self.template_name)
  File "/usr/local/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.5/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.5/site-packages/jinja2/loaders.py", line 187, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: jinja2_report.html
```

Reply:
Yes, my bad, I forgot to add template file to vcs